src/main/java/nl/tudelft/ewi/devhub/server/database/entities/issues/AbstractIssue.java

```diff
@@ -71,6 +77,18 @@
 	@Temporal(TemporalType.TIMESTAMP)
 	private Date closed;
 
+	// Left away @NotEmpty for now, since existing tests should be updated for it to work
```

Comment:
I have some small issues implementing this for pull-requests. The point is that the title must be present at creation time, but the id is not generated then. This gives a typical "Chicken or the egg" problem. We could place some dummy content, and update is directly after persisting, but that seems a bit hackish to me. Do you have an idea about how to fix that cleanly?